nextflow_schema.json

```
@@ -828,6 +828,63 @@
                     "help_text": "Specifies what the maximum misincorporation frequency should be displayed as, in the DamageProfiler damage plot. This is set to 0.30 (i.e. 30%) by default as this matches the popular [mapDamage2.0 program](https://ginolhac.github.io/mapDamage/). However, the default behavior of the DamageProfiler is to 'autoscale' the y-axis maximum to zoom in on any possible damage that may occur (e.g. if the damage is about 10%, the highest value on the y-axis would be set to 0.12). This 'autoscale' behavior can be turned on by specifying the number to 0. Default: 0.30.\n\n> Modifies DamageProfile parameter: -yaxis_dp_max"
                 }
             }
+        },
+        "contamination_estimation": {
+            "title": "Contamination estimation",
+            "type": "object",
+            "description": "Options for the estimation of contamination",
+            "default": "",
+            "fa_icon": "fas fa-radiation-alt",
+            "properties": {
+                "run_contamination_estimation_angsd": {
+                    "type": "boolean",
+                    "description": "Turn on nuclear contamination estimation for genomes with ANGSD.",
+                    "help_text": "Specify to run the optional processes for nuclear DNA contamination estimation with ANGSD.",
+                    "fa_icon": "fas fa-power-off"
+                },
+                "contamination_estimation_angsd_chrom_name": {
+                    "type": "string",
+                    "default": "X",
+                    "description": "The name of the chromosome to be used for contamination estimation.",
+                    "help_text": "The name of the chromosome as specified in your FASTA/bam header.\ne.g. 'X' for hs37d5, 'chrX' for HG19",
+                    "fa_icon": "fas fa-address-card"
+                },
+                "contamination_estimation_angsd_range_pos1": {
+                    "type": "integer",
+                    "default": 5000000,
+                    "description": "The first position on the chromosome to be used for contamination estimation with ANGSD.",
+                    "help_text": "The beginning of the genetic range that should be utilised for nuclear contamination estimation. ",
+                    "fa_icon": "fas fa-map-marker-alt"
+                },
+                "contamination_estimation_angsd_range_pos2": {
```

```suggestion
                "contamination_estimation_angsd_range_from": {
                    "type": "integer",
                    "default": 5000000,
                    "description": "The first position on the chromosome to be used for contamination estimation with ANGSD.",
                    "help_text": "The beginning of the genetic range that should be utilised for nuclear contamination estimation. ",
                    "fa_icon": "fas fa-map-marker-alt"
                },
                "contamination_estimation_angsd_range_to": {
```
Though I suggest using the schema editor instead of this